paasta_tools/utils.py

```diff
@@ -857,6 +857,39 @@ def get_cluster_fqdn_format(self):
         :returns: A format string for constructing the FQDN of the masters in a given cluster."""
         return self.get('cluster_fqdn_format', 'paasta-{cluster:s}.yelp')
 
+    def get_chronos_config(self):
+        """Get the chronos config
+
+        :returns: The chronos config dictionary"""
+        try:
+            return self['chronos_config']
+        except KeyError:
+            raise PaastaNotConfiguredError(
```

Comment:
I was just following what other `get_x` functions were doing in `SystemPaastaConfig`.